### ISSUE TITLE
Still being worked on?

### ISSUE THREAD

**Issue by gregmax17** [chandlerprall/GoblinPhysics#72]
Is this still being worked on? It looks really promising!

I like how simple it is to get this up and running with three js. 

Can I up vote `More event callbacks` feature please, thanks!

ps - Why don't the Tests work?

**Comment by dark-swordsman** [comment]
Life may have gotten in the way, lol. I am really interested in this too. The tests even ran amazing on my Note 4 easily exceeding 60 FPS.

**Comment by chandlerprall** [comment]
Indeed. Life, family, work, etc. Also, the library is very stable at this point, though there's an outstanding feature branch with more constraints I never got to finishing.

Largely, the library is sufficient for my needs and it didn't gain additional users (that I saw). I'm around to support Goblin, but adding more features is mostly off the table.
